src/components/Footer.svelte

```diff
@@ -0,0 +1,114 @@
+<script lang="ts">
+  export let semesters: string[];
+</script>
+
+<footer class="row-center">
+  <div id="footer-contents">
+    <div id="inquiries">
+      <div id="logo-container">
+        <a rel="prefetch" href="."
+          ><img src="/logo.svg" alt="Hack4Impact logo" /></a
+        >
+      </div>
+
+      For all inquiries of partnership or sponsorship, please contact us at
+      <a class="blue-text" href="mailto:uiuc@hack4impact.org"
+        >uiuc@hack4impact.org</a
+      >
+    </div>
+    <div class="links">
+      <div class="link-section">
+        <span class="blue-text link-section-header">Connect With Us</span>
+        <a href="https://www.facebook.com/h4iuiuc">Facebook</a>
+        <a href="https://github.com/hack4impact-uiuc">GitHub</a>
+
+        <a href="https://www.instagram.com/hack4impactuiuc/">Instagram</a>
+      </div>
+      <div class="link-section">
+        <a class="blue-text link-section-header" href="/about">About Us</a>
```

Comment:
This goes for all of the links, but using unordered lists for the links would be more semantic